src/App.tsx

```diff
@@ -10,9 +32,17 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<h1 className="title">Home Page</h1>} />
+            <Route path="/home" element={<Navigate to="/" replace />} />
+            <Route path="/people" element={<PeoplePage />}>
+              <Route path="/people/:slug" element={<PeoplePage />} />
+            </Route>
```

Comment:
The nested <Route> inside the '/people' route seems unnecessary if it is intended to render the same component, PeoplePage. If you want to render a different component or a specific part of PeoplePage for '/people/:slug', consider using a different component or handling the logic within PeoplePage.